controllers/change_coordinators.go

```diff
@@ -185,3 +186,18 @@ func selectCoordinators(cluster *fdbtypes.FoundationDBCluster, status *fdbtypes.
 
 	return coordinators, nil
 }
+
+func getCoordinatorAddress(cluster *fdbtypes.FoundationDBCluster, info localityInfo) fdbtypes.ProcessAddress {
```

Comment:
```suggestion
func getCoordinatorAddress(cluster *fdbtypes.FoundationDBCluster, info localityInfo) fdbtypes.ProcessAddress {
```

locality might be a better name here than `info` but I'm open to both names.